pyproject.toml

```diff
@@ -13,6 +13,8 @@ importlib-metadata = {version = "^4.8.1", python = "<3.8"}
 click = "^8.0.3"
 uvicorn = "^0.15.0"
 PyYAML = "^6.0"
+alembic = "^1.7.5"
+psycopg2-binary = "^2.9.2"
```

Comment:
Let's use plain `psycopg2` here. I know about the warnings that installing it with plain `pip` (probably `poetry` too) yields, but we have `psycopg2` satisfied as an RPM package in Fedora and I suppose CentOS, too.